### PR TITLE
Add support for Azure, OpenAI, Palm, Anthropic, Cohere Models - using litellm

### PR DESCRIPTION
**Pull request by ishaan-jaff**
I'm the maintainer of litellm https://github.com/BerriAI/litellm - a simple & light package to call OpenAI, Azure, Cohere, Anthropic API Endpoints

This PR adds support for models from all the above mentioned providers

Here's a sample of how it's used:

```
from litellm import completion

## set ENV variables
# ENV variables can be set in .env file, too. Example in .env.example
os.environ["OPENAI_API_KEY"] = "openai key"
os.environ["COHERE_API_KEY"] = "cohere key"

messages = [{ "content": "Hello, how are you?","role": "user"}]

# openai call
response = completion(model="gpt-3.5-turbo", messages=messages)

# cohere call
response = completion("command-nightly", messages)

# anthropic call
response = completion(model="claude-instant-1", messages=messages)
```

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

File: requirements-base.txt [hunk 1]
```diff
@@ -109,3 +109,5 @@ phabricator>=0.7.0
 selenium>=4.1.5
 
 openai==0.27.0
+
+litellm==0.1.213
```
Comment (asottile-sentry) [inline requirements-base.txt:113]: this pulls in a bunch of dependencies that we don't use

**Review by asottile-sentry — CHANGES_REQUESTED** [review]

**Comment by asottile-sentry** [comment]
oh also this library only supports python 3.9+ so we wouldn't be able to use it anyway

File: requirements-base.txt (hunk 1 above)
Comment (ishaan-jaff) [reply requirements-base.txt:113]: is there somewhere else I should add this ? @asottile-sentry 

If you're not trying to add multiple llm providers i'm assuming this is not useful

Comment (asottile-sentry) [reply requirements-base.txt:113]: I don't believe we're trying to add multiple llm providers

**Pull request closed without merging**